entrypoint.sh

```diff
@@ -579,6 +580,24 @@ function start_rgw {
   fi
 }
 
+function create_rgw_user {
+
+  # Check to see if our RGW has been initialized
+  if [ ! -e /var/lib/ceph/radosgw/keyring ]; then
+    echo "ERROR- /var/lib/ceph/radosgw/keyring must exist. You can extract it from your current monitor by running 'ceph auth get client.bootstrap-rgw -o /var/lib/ceph/bootstrap-rgw/${CLUSTER}.keyring'"
```

Comment:
Weird we are checking for `/var/lib/ceph/radosgw/keyring` and then recommending to store it in  `/var/lib/ceph/bootstrap-rgw/${CLUSTER}.keyring`.